lib/git_fame/base.rb

```diff
@@ -261,12 +264,19 @@ def convert_exclude_paths_to_array
     end
 
     #
+    # Converts @extensions argument to an array
+    #
+    def convert_extensions_to_array
+      @extensions.split(",")
+    end
```

Comment:
Don't worry about it. I started cleaning up the code base a few days back (haven't had the time to push yet) removing methods like this. You can keep it and I'll refactor it later on.